cmd/parse/blocks/missing.go

```diff
@@ -0,0 +1,52 @@
+package blocks
+
+import (
+	"fmt"
+	"strconv"
+
+	parsecmdtypes "github.com/forbole/juno/v4/cmd/parse/types"
+
+	"github.com/spf13/cobra"
+
+	"github.com/forbole/juno/v4/parser"
+	"github.com/forbole/juno/v4/types/config"
+)
+
+// newMissingCmd returns a Cobra command that allows to fix missing blocks in database
+func newMissingCmd(parseConfig *parsecmdtypes.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "missing [start height]",
+		Short: "Refetch all the missing heights in the database starting from the given start height",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			startHeight, err := strconv.ParseInt(args[0], 10, 64)
+			if err != nil {
+				return fmt.Errorf("make sure the given start height is a positive integer")
+			}
+
+			parseCtx, err := parsecmdtypes.GetParserContext(config.Cfg, parseConfig)
+			if err != nil {
+				return err
+			}
+
+			workerCtx := parser.NewContext(parseCtx.EncodingConfig, parseCtx.Node, parseCtx.Database, parseCtx.Logger, parseCtx.Modules)
+			worker := parser.NewWorker(workerCtx, nil, 0)
+
+			DbLastHeight, err := parseCtx.Database.GetLastBlockHeight()
```

Comment:
Please use camelCase instead of PascalCase when naming variables
```suggestion
			dbLastHeight, err := parseCtx.Database.GetLastBlockHeight()
```